src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -598,13 +598,13 @@ class PolicyHandler : public PolicyHandlerInterface,
    * @param new_app_id app_id for this application
    * @param policy_id policy_id for this application
    */
-  void OnAddedNewApplicationToAppList(const uint32_t new_app_id,
-                                      const std::string policy_id) OVERRIDE;
+  void OnAddedNewApplicationToAppList(const uint32_t& new_app_id,
```

Comment:
@ydementieiev there is no need to pass int as a reference

Reply:
@AKalinich-Luxoft done [f0f9ced](https://github.com/smartdevicelink/sdl_core/pull/3208/commits/f0f9ced5f7902b7d25d333c8d1559a7e3f374f89)